node/src/components/consensus/highway_core/state/tests.rs

```diff
@@ -173,11 +174,16 @@ fn add_vote() -> Result<(), AddVoteError<TestContext>> {
         .err()
         .map(vote_err);
     assert_eq!(Some(VoteError::InconsistentPanorama(BOB)), opt_err);
-    // And you can't change the round exponent within a round.
+    // And you can't make the round exponent too small
```

Comment:
This was actually testing something else – that you cannot use different round exponent for confirmation/witness vote, only for the proposal. Although it indeed used `VoteError::RoundLength` for that. Let's see what @afck has to say about it.

Reply:
Yes: Bob's round length is 16, and he's changing it to 32 in the next line, at time 50.
However, his previous vote `b1` was at time `49`, and there is no `49 < t ≤ 50` that is divisible by 32.

We should make `b1` above more explicit:
```rust
let b1 = add_vote!(state, rng, BOB, 49, 4u8, None; N, b0, c0)?;
```

Reply:
Fixed in https://github.com/CasperLabs/casper-node/pull/479/commits/0fca00cb55e7da746b6ffb954b020fd8d1ee9e22